test/integration.test.js

```diff
@@ -17,19 +17,19 @@ describe("Integration Test", function () {
   it("should verify that the entire process works", async function () {
 
     const maxTotalSupply = 10000000000; // 10 billions
-    let [owner, user1, user2, user3] = await ethers.getSigners();
+    let [deployer, owner, user1, user2, user3] = await ethers.getSigners();
     const SSYN = await ethers.getContractFactory("EscrowedSyndicateERC20");
-    const ssyn = await SSYN.deploy();
+    const ssyn = await SSYN.deploy(owner.address);
     const SYN = await ethers.getContractFactory("SyndicateERC20");
-    const syn = await SYN.deploy(owner.address, maxTotalSupply);
+    const syn = await SYN.deploy(owner.address, maxTotalSupply, owner.address);
```

Comment:
We said that it is better if we test with two different accounts for the token owner and the contract owner.